src/message/messageActionSheet.js

```diff
@@ -232,15 +232,19 @@ export const showActionSheet = (
   const optionCodes = isHeader
     ? constructHeaderActionButtons(params)
     : constructMessageActionButtons(params);
-  const callback = buttonIndex => {
-    allButtons[optionCodes[buttonIndex]]({
-      dispatch,
-      subscriptions: params.backgroundData.subscriptions,
-      auth: params.backgroundData.auth,
-      ownEmail: params.backgroundData.ownEmail,
-      _,
-      ...params,
-    });
+  const callback = async buttonIndex => {
+    try {
+      await allButtons[optionCodes[buttonIndex]]({
+        dispatch,
+        subscriptions: params.backgroundData.subscriptions,
+        auth: params.backgroundData.auth,
+        ownEmail: params.backgroundData.ownEmail,
+        _,
+        ...params,
+      });
+    } catch (err) {
+      Alert.alert('Couldnt perform the requested action', err.message);
```

Comment:
Typo: `Couldn't` (although preferentially `Could not`).

That said, it's probably worth the trouble to do better than a generic `Could not perform the requested action` here. We can associate a failure message with each action (`Could not delete message`, etc.) the same way we're associating a title.

Reply:
I have replaced the message with a simpe 'Error'. This will not be confusing because the alert message contains context-specific detailed message.

Reply:
Counterpoint:

![Screenshot_20200116_130132](https://user-images.githubusercontent.com/54861490/72562458-58706c00-3860-11ea-83b0-1bca83897c8b.png)